src/views/index.html

```diff
@@ -77,6 +77,7 @@
     <nav id="main-menu" class="hidden">
       <ul>
         <li><a id="things-menu-item" href="/things" class="selected">Things</a></li>
+        <li><a id="rules-engine-menu-item" href="/rules-engine">Rules Engine</a></li>
```

Comment:
We could add a checkbox in an "experiments" section of settings which defaults to off https://docs.google.com/presentation/d/1rMeg6hep8JQflqwbiNIbhSoK9StCeVZ2npphmQHKda4/edit#slide=id.g24e68acc65_0_9 which could eventually become an add-ons section.

Ideally the icon would be provided by the rules engine add-on itself, presumably an add-on would specify it in a manifest a bit like web extensions add buttons to the toolbar https://developer.mozilla.org/en-US/Add-ons/WebExtensions/user_interface/Browser_action